src/types/Promotion.ts

```diff
@@ -0,0 +1,36 @@
+import {RFC33339Date} from './BapiProduct';
+
+export interface PromotionCondition {
+  level: 'item' | 'global';
+  key: string;
+  condition: string;
+}
+
+export type PromotionEffect =
+  | {
+      type: 'automatic_discount';
+      additionalData: {
+        type: 'absolute' | 'relative';
```

Comment:
@HenriBeck One minor thing here. In response we get `percentage` type which is not defined here (that should instead of `relative` I guess). So either we should modify the response payload or change the types here

Reply:
You are totally right. 

I'll let my team change this since `absolute` | `relative` is a bit more consistent with the rest of the APIs.